pkgs/development/libraries/openbabel/builder_python_bindings_enabled.sh

```diff
@@ -0,0 +1,30 @@
+source $stdenv/setup
+mkdir $out
+tar xvfz $src
+cd openbabel-*
+mkdir ob-build
+cd ob-build
+cmake -DRUN_SWIG=ON -DPYTHON_BINDINGS=ON -DCMAKE_INSTALL_PREFIX=$out -DLIB_INSTALL_DIR=$out  ..
+make -j$NIX_BUILD_CORES install
+patchelf --add-needed $out/libopenbabel.so.7 $out/lib/python$pythonMajorMinor/site-packages/openbabel/_openbabel.so
+
+cd $out/bin
+for file in `ls`; do
+    patchelf --add-needed $out/libopenbabel.so.7 $file
```

Comment:
Can we put this into the package into the correct phases?

Reply:
OK, i grouped things into build stages in the shell script, but i'm still reading about how to use the build phase attributes correctly

Reply:
done!